File: benchmark/results/java-invocation-inline/run-001/b-baseline/output/com/metaengine/demo/shipping/services/CarrierGateway.java
package com.metaengine.demo.shipping.services;

import com.metaengine.demo.shipping.aggregates.Shipment;
import java.util.List;

/** Gateway service that brokers shipments through external carriers. */
public class CarrierGateway {
    /** Submit a new shipment from the provided partial input. */
    public Shipment create(Shipment input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Look up a brokered shipment by id. Returns null when not found. */
    public Shipment findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List up to {@code limit} brokered shipments. */
    public List<Shipment> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Cancel a brokered shipment by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/ts-script-sonnet/run-005/a-mcp/output/src/domain/shipping/services/carrier-gateway.ts
import { Shipment } from '../aggregates/shipment';

export class CarrierGateway {

  create(input: Partial<Shipment>): Shipment { throw new Error('not implemented'); }

  findById(id: string): Shipment | null { throw new Error('not implemented'); }

  list(limit: number): Shipment[] { throw new Error('not implemented'); }

  delete(id: string): void { throw new Error('not implemented'); }
}
